worlds/apquest/test/test_hammer.py

```diff
@@ -0,0 +1,59 @@
+from typing import cast
+
+from test.param import classvar_matrix
+
+from ..items import APQuestItem
+from .bases import APQuestTestBase
+
+# This file assumes that you've read test_extra_starting_chest.py first.
+# classvar_matrix will not be explained again here, so make sure you've read that first.
+
+# We'll once again use classvar_matrix to test that the world behaves correctly both if "hammer" option is disabled,
+# and also that it behaves correctly when the "hammer" option is enabled.
+hammer_options = [
+    {"hammer": False},
+    {"hammer": True},
+]
+
+
+@classvar_matrix(options=hammer_options)
```

Comment:
So, I understand demonstrating classvar matrix, even though APQuest is too simple to really benefit from it. But why are we doing that _twice_? If this file isn't showing us anything new about matrix tests, wouldn't it be simpler to do four normal tests?

Reply:
Maybe you're right. I just straight up prefer using classvar_matrix always since learning about it, but I think that's my personal bias.

Reply:
In Witness, I use a Multiworld test for this. I *really* like having "yes" and "no" in the same class, rather than having a TestYes class and a TestNk class

Reply:
Yeah, we probably all have some personal things we like doing in our test organization. Like how I define `assertReachableWith()` and `assertNotReachableWith()` helpers in my test base instead of trying to puzzle out whether `assertAccessDependency()` means what I want it to mean. But I wouldn't put those in APQuest tests aimed at complete novices, unless AP core decided to turn those into "standard" world test helpers first.

Reply:
I think I'm just gonna entirely get rid of classvar_matrix then